src/main/java/edu/iis/powp/adapter/PlotterAdapter.java

```diff
@@ -8,11 +8,12 @@
 /**
  * Plotter adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements IPlotter {
+public class PlotterAdapter implements IPlotter {

```

Comment:
The name is better but it does not indicate the purpose of the adapter or what was actually adapted.